src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -131,22 +140,22 @@ var topLevelEventsToDispatchConfig = {};
 
 var onClickListeners = {};
 
-function getDictionaryKey(inst) {
+function getDictionaryKey(inst: ReactInstance): string {
   // Prevents V8 performance issue:
   // https://github.com/facebook/react/pull/7232
   return '.' + inst._rootNodeID;
 }
 
-var SimpleEventPlugin = {
+var SimpleEventPlugin: PluginModule<MouseEvent> = {
 
   eventTypes: eventTypes,
 
   extractEvents: function(
-    topLevelType,
-    targetInst,
-    nativeEvent,
-    nativeEventTarget
-  ) {
+    topLevelType: TopLevelTypes,
+    targetInst: ReactInstance,
+    nativeEvent: MouseEvent,
+    nativeEventTarget: EventTarget,
+  ): null | ReactSyntheticEvent {
```

Comment:
Ignore me, I guess `?ReactSyntheticEvent` would also include undefined and I guess you aren't wanting that.